changelogs/unreleased/4684-keep-requirements-v1tov2.yml

```
@@ -0,0 +1,6 @@
+description: "Fix issue where the v1tov2 command removes the requirements.txt file"
+issue-nr: 4684
+change-type: patch
+destination-branches: [master, iso5]
+sections:
+  bugfix: "{{description}}"
```

```suggestion
  minor-improvement: "{{description}}"
```
It doesn't cause any real issues, it's just not in line with Python conventions. I think we might confuse users if we mark this as a bug.